dynalab/tasks/flores_small1.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+import uuid
+
+from dynalab.tasks.common import BaseTaskIO
+
+
+data = [
+    {
+        "uid": str(uuid.uuid4()),
+        "sourceLanguage": "en_XX",
+        "targetLanguage": "hr_HR",
+        "sourceText": "Hello world !",
+    },
+    {
+        "uid": str(uuid.uuid4()),
+        "sourceLanguage": "hr_HR",
+        "targetLanguage": "en_XX",
+        "sourceText": "Toni Kukoč sjajan je košarkaš.",
+    },
+]
+
+
+class TaskIO(BaseTaskIO):
+    def __init__(self):
+        BaseTaskIO.__init__(self)
+
+    def verify_response(self, response, data):
+        assert "id" in response and response["id"] == data["uid"]
+        assert "translatedText" in response and isinstance(
+            response["translatedText"], str
+        )
+        assert response["signed"] == self.generate_response_signature(response, data)
```

Comment:
For security purpose, please follow the convention in other tasks and check there are no extra keys too.

Reply:
done.